main.py

```diff
@@ -10,21 +9,18 @@
 import netCDF4  # for strange reasons, one has to import netCDF4 before geopyspark on some systems, otherwise reading nc-files fails.
 import geopyspark as gps
 from pyspark import SparkContext
-from dotenv import load_dotenv
-
+from metadata_schema import *
 from geoPy import geopy
 
-load_dotenv()
```

Comment:
Why move this to `metadata_schema.py`? This loads env variables not only related to the database, so we should at least have it here also, I'd say

Reply:
`metadata_schema.py` also needs to load env to init the database connection.

Reply:
I see. Still, I don't like how .env is only loaded implicitly in main. 
How about this refactoring: Create a module `settings.py` that executes `load_dotenv()` and reads the env variables into constants. Then in `main.py` and `metadata_schema.py` you can import these constants as needed.

Reply:
We can directly call `load_dotenv()` in `main.py` as well, not a big deal I think.

Reply:
Yes, but I find it cleaner to have a single point that loads all settings. Especially when there might come up other modules in the future that also need settings. Also, it makes switching to other configuration management tools easier because we don't use `os.getenv` all around our modules.

Reply:
Sure, we can go this way.